Dockerfile

```diff
@@ -10,6 +10,7 @@ FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
 COPY --from=builder /tmp/build/cluster-network-operator /usr/bin/
 COPY --from=builder /tmp/build/cluster-network-renderer /usr/bin/
 COPY manifests /manifests
+COPY vendor/github.com/openshift/api/operator/v1/0000_70_cluster-network-operator_01_crd.yaml /manifests
```

Comment:
Diffing the old and new, it looks like we are losing the `required` tags (which should be set on `spec.defaultNetwork`, `spec.clusterNetwork[].cidr`, `spec.clusterNetwork[].hostPrefix`, `spec.defaultNetwork.type`, `spec.additionalNetworks[].type`, and `spec.additionalNetworks[].name`)

Reply:
thanks @danwinship. I have addressed the review comments. Looks like we need to add more annotations for these required fields in https://github.com/openshift/api/pull/527